messaging/src/main/java/org/axonframework/eventhandling/TrackingEventProcessor.java

```diff
@@ -864,6 +865,18 @@ public Throwable getError() {
             return errorState;
         }
 
+        @Override
+        public OptionalLong getReplayPosition() {
```

Comment:
Based on the discussions we've had, I think this should be replaced for a `getPosition()`.

Reply:
As said up, the exposed method name is getCurrentPosition()